test/merchant_repository_test.rb

```diff
@@ -0,0 +1,25 @@
+require './test/test_helper'
+require 'csv'
+require './lib/merchant_repository'
+require 'pry'
+
+class MerchantRepositoryTest < Minitest::Test
+  def setup
+    file = './data/test_merchant.csv'
+    @repo = MerchantRepository.new(file)
+  end
+
+  def test_it_exists
+    assert_instance_of MerchantRepository, @repo
+  end
+
+  def test_build_merchants
+    @repo.build_merchants
+    assert_equal 2, @repo.merchants.count
+  end
+
+  def test_merchants
```

Comment:
@Jenny this method and the `test_build_merchants` are the same -- could they be consolidated, or is there something separate to test in each method which could be more explicit?

Reply:
(Was SimpleCov needing this extra method to get to 100%?)

Reply:
Was able to remove the merchants test and it still ran as expected with 100% on SimpleCov